r599 | ph10 | 2011-05-07 18:09:06 +0200 (So, 07 kvÄ› 2011) | 2 lines

Fix typos in pcregrep and pcretest man pages.
http://bugs.exim.org/show_bug.cgi?id=1081

Petr Pisar: Changelog lines removed


Index: doc/pcretest.1
===================================================================
--- doc/pcretest.1	(revision 598)
+++ doc/pcretest.1	(revision 599)
@@ -25,8 +25,8 @@
 .rs
 .TP 10
 \fB-b\fP
-Behave as if each regex has the \fB/B\fP (show bytecode) modifier; the internal
-form is output after compilation.
+Behave as if each regex has the \fB/B\fP (show byte code) modifier; the
+internal form is output after compilation.
 .TP 10
 \fB-C\fP
 Output the version number of the PCRE library, and all available information
@@ -76,7 +76,7 @@
 Do not output the version number of \fBpcretest\fP at the start of execution.
 .TP 10
 \fB-S\fP \fIsize\fP
-On Unix-like systems, set the size of the runtime stack to \fIsize\fP
+On Unix-like systems, set the size of the run-time stack to \fIsize\fP
 megabytes.
 .TP 10
 \fB-t\fP
@@ -154,7 +154,7 @@
 A pattern may be followed by any number of modifiers, which are mostly single
 characters. Following Perl usage, these are referred to below as, for example,
 "the \fB/i\fP modifier", even though the delimiter of the pattern need not
-always be a slash, and no slash is used when writing modifiers. Whitespace may
+always be a slash, and no slash is used when writing modifiers. White space may
 appear between the final pattern delimiter and the first modifier, and between
 the modifiers themselves.
 .P
@@ -327,8 +327,8 @@
 .rs
 .sp
 Before each data line is passed to \fBpcre_exec()\fP, leading and trailing
-whitespace is removed, and it is then scanned for \e escapes. Some of these are
-pretty esoteric features, intended for checking out some of the more
+white space is removed, and it is then scanned for \e escapes. Some of these
+are pretty esoteric features, intended for checking out some of the more
 complicated features of PCRE. If you are just testing "ordinary" regular
 expressions, you probably don't need any of these. The following escapes are
 recognized:
@@ -336,7 +336,7 @@
   \ea         alarm (BEL, \ex07)
   \eb         backspace (\ex08)
   \ee         escape (\ex27)
-  \ef         formfeed (\ex0c)
+  \ef         form feed (\ex0c)
   \en         newline (\ex0a)
 .\" JOIN
   \eqdd       set the PCRE_MATCH_LIMIT limit to dd
@@ -712,7 +712,7 @@
 .rs
 .sp
 The facilities described in this section are not available when the POSIX
-inteface to PCRE is being used, that is, when the \fB/P\fP pattern modifier is
+interface to PCRE is being used, that is, when the \fB/P\fP pattern modifier is
 specified.
 .P
 When the POSIX interface is not in use, you can cause \fBpcretest\fP to write a
@@ -736,7 +736,7 @@
 follows immediately after the compiled pattern. After writing the file,
 \fBpcretest\fP expects to read a new pattern.
 .P
-A saved pattern can be reloaded into \fBpcretest\fP by specifing < and a file
+A saved pattern can be reloaded into \fBpcretest\fP by specifying < and a file
 name instead of a pattern. The name of the file must not contain a < character,
 as otherwise \fBpcretest\fP will interpret the line as a pattern delimited by <
 characters.
Index: doc/pcregrep.1
===================================================================
--- doc/pcregrep.1	(revision 598)
+++ doc/pcregrep.1	(revision 599)
@@ -356,7 +356,7 @@
 which recognizes any of the preceding three types, and an "any" convention, in
 which any Unicode line ending sequence is assumed to end a line. The Unicode
 sequences are the three just mentioned, plus VT (vertical tab, U+000B), FF
-(formfeed, U+000C), NEL (next line, U+0085), LS (line separator, U+2028), and
+(form feed, U+000C), NEL (next line, U+0085), LS (line separator, U+2028), and
 PS (paragraph separator, U+2029).
 .sp
 When the PCRE library is built, a default line-ending sequence is specified.
@@ -530,9 +530,9 @@
 .rs
 .sp
 Exit status is 0 if any matches were found, 1 if no matches were found, and 2
-for syntax errors and non-existent or inacessible files (even if matches were
+for syntax errors and non-existent or inaccessible files (even if matches were
 found in other files) or too many matching errors. Using the \fB-s\fP option to
-suppress error messages about inaccessble files does not affect the return
+suppress error messages about inaccessible files does not affect the return
 code.
 .
 .
